inspection/contentScript.js

```diff
@@ -582,12 +583,7 @@ function sendInspectInfo(command, event) {
 	}
 
 	// ToDo: for payload create user define datatype
-	const payload = {
-		cmd: 'inspecting',
-		value: data,
-	};
-
-	console.log(payload);
+	const payload = new stepObject('inspecting', data);
```

Comment:
let's revert this back to previous code for now:
```
const payload = {
		cmd: 'inspecting',
		value: data,
	};
```
We need to rework this function entirely at a later time

Reply:
Reverted the current changes with above code.